src/Symfony/Component/BrowserKit/Response.php

```diff
@@ -87,4 +90,23 @@ public function getHeader(string $header, bool $first = true): string|array|null
 
         return $first ? null : [];
     }
+
+    public function toArray(): array
+    {
+        if (isset($this->jsonData)) {
+            return $this->jsonData;
```

Comment:
@rodnaph I doubt it makes much of a difference either way, the reason I added it is because the `toArray` method from the HttpClient component has it: https://github.com/symfony/symfony/blob/25a5eb20b5fbe3853579d0e634e3aa987a7229da/src/Symfony/Component/HttpClient/Response/CommonResponseTrait.php#L81-L90
Then again, now that I look at it, the one in the HttpClient component looks odd, shouldn't the caching go before the call to `getContent`? cc @nicolas-grekas 

In any case, I have no strong opinion about this.

Reply:
The code on HttpClient is on purpose.

For the one on BrowserKit, I'm fine either way. Holding the value in memory for the duration of the response object shouldn't be an issue I think.